Home_work12/Home_work12.cpp

```diff
@@ -0,0 +1,93 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+#include <iostream>
+#include "Save.h"
+#include <map>
+
+
+class WordBase {
+    std::ifstream file;
+public:
+    std::string word(int num) {
+        std::map <int, std::string> words;
+        file.open("words.txt", std::ios::in);
+        
+        for (int i = 0; std::getline(file, words[i]); ++i);
+        file.close();
+        std::srand(num);
+        int generatingRange = words.size();
+        int generatedNum = std::rand() % (generatingRange - 1);
+
+        return words[generatedNum];
+    }
+};
+
+class Game {
+    static const int wordSize = 6;
+    char quess[wordSize]{};
+    int counter{};
+    char result[wordSize]{};
+public:
+    Game( ){
+        
+    }
+    void game(std::string word) {
+        while (true) {
+            std::cout << "Enter: ";
+            std::cin >> quess;
+            for (int i = 0; i < wordSize-1; i++) {
+                result[i]= '*';
+                if (quess[i] == word[i]) {
+
+                    result[i] = std::toupper(quess[i]);
+                }
+                else {
+                    for (int j = 0; j < wordSize-1; j++) {
+                        if (quess[i] == word[j]) {
+                            result[i] = quess[i];
```

Comment:
Don't sure it will work for all edge-cases
e.g.: `books`, `cocon`, `woman` from both sides